firefox-ios/Client/Frontend/Browser/Toolbars/Redux/ToolbarState.swift

```diff
@@ -10,6 +10,7 @@ struct ToolbarState: ScreenState, Equatable {
     var windowUUID: WindowUUID
     var toolbarPosition: AddressToolbarPosition
     var isPrivateMode: Bool
+    var isLoginActionRequired: Bool
```

Comment:
We shouldn't need to save this in the state as this is already saved through having a `badgeImageName` on the `ToolbarActionState` or not.